tamr_toolbox/dataset/manage.py

```diff
@@ -0,0 +1,225 @@
+from typing import List, Optional, Union
+
+from tamr_unify_client import Client
+from tamr_unify_client.dataset.resource import Dataset
+from tamr_unify_client.attribute.resource import AttributeSpec
+from tamr_unify_client.attribute.type import AttributeType
+
+from tamr_toolbox.models.data_type import JsonDict
+
+
+def create(
+    *,
+    tamr: Client,
```

Comment:
Change to client: Client for consistency with toolbox modules
```suggestion
    client: Client,
```

Reply:
Thanks! Updated